src/auth.py

```diff
@@ -107,6 +114,16 @@ def rq_get(request: Request, key: str) -> str:
         return j.get(key, "")
     return ""
 
+def get_facebook_public_key():
```

Comment:
Add an `@lrucache` or `@cache` decorator here - we don't need to re-fetch this on every login